.github/workflows/ci_back_build_and_test.yml

```diff
@@ -5,9 +5,20 @@ name: Back-end build and test
 
 on:
   push:
```

Comment:
TBH it doesn't really matter AFAIK
If it is triggering workflow on pushes it doesn't matter as branch protection policies prohibit us from pushing directly to pushing directly to either master or dev

Reply:
Unless it also detects GitHub pushes, but we still do those od PRs so why do them again, on exactly the same code after approve?

Reply:
I think it is better to have them rather than not to have them. In rather odd cases we might want a workflow run on master.